### ISSUE TITLE
Renaming player doesn't open keyboard

### ISSUE THREAD

**Issue by andiweli** [fgsfdsfgs/UE1#8]
First of all: The port I've waited for 🥹 Superb work!

When trying to start a game and renaming the "Player 1" no keyboard shows up. 

Thanks for this port 👍🏻

PS: An Android port would be great too.

**Comment by fgsfdsfgs** [comment]
Yeah, there's no on screen keyboard support yet. You can change the name in `System/Unreal.ini` somewhere for now.

**Comment by andiweli** [comment]
> Yeah, there's no on screen keyboard support yet. You can change the name in `System/Unreal.ini` somewhere for now.

I already thought of that :D Thanks.

What about an Android port of the game? Do you think it is possible?

**Comment by fgsfdsfgs** [comment]
Probably, for ARMv7 devices at least. I've never made anything for Android.

**Comment by andiweli** [comment]
> Probably, for ARMv7 devices at least. I've never made anything for Android.

That would be great :)
